mpstat.c

```diff
@@ -305,7 +306,14 @@ void write_irqcpu_stats(struct stats_irqcpu *st_ic[], int ic_nr, int dis,
 				if (!strcmp(p0->irq_name, q0->irq_name) || !interval) {
 					p = st_ic[curr] + (cpu - 1) * ic_nr + j;
 					q = st_ic[prev] + (cpu - 1) * ic_nr + offset;
-					printf(" %10.2f",
+					/* Width is IRQ name + 2 for the /s */
+					colwidth=strlen(p0->irq_name)+2;
```

Comment:
Instead of calculating it each time, while we are printing the header, couldn't we save the lengths in an array that we use here, with the minimal calculations from lines 313 and 314 below?

Maybe it is not worth it, as this is not really a performance issue, but it might make it clearer to other readers in the future.

Also, you might want to take a look at the code around this addition and get this addition to match that styling (spaces after commas, before and after operators, etc.).